mrr/src/main/java/gov/cdc/dex/mrr/RedisUtility.kt

```diff
@@ -9,8 +9,7 @@ class RedisUtility {
         val redisCacheName = System.getenv("REDISCACHEHOSTNAME")
         val rediscachekey = System.getenv("REDISCACHEKEY")
         var jedis: Jedis? = null;
-        // rdisCacheName = "temedehl7.redis.cache.windows.net"
-        // rediscachekey ="1ofYheE07YlzGOuDAauSQwQ09tj5u4fVrAzCaKAsQt0="

```

Comment:
Remove password even from comments.
-Also, you are swallowing the exception here, which is the issue with #114 
- Change println() to a logger (we've been using slf4j).
- after logging the exception, throw it back.
- then change the return value from Jedis? (optional) to Jedis - always not null -> this will improve the code using this method.

Reply:
done.